### ISSUE TITLE
Periodically gets stuck and repeats a key a bunch of times

### ISSUE THREAD

**Issue by szbergeron** [tadosappo/nasskan#7]
Not sure how to help get any debug/log info for this.

Periodically I'll type some key and it will not send a keyup event, resulting in that key getting repeated until I press some other key, which seems to interrupt it.

**Comment by tadosappo** [comment]
yeah I sometimes encounter this issue. I was unable to confirm the steps to reproduce this issue...

**Comment by szbergeron** [comment]
I can reliably get this to happen by manually `systemctl start`ing it, it ends up repeating the enter key after I hit enter after putting in my password.
